configure.ac

```diff
@@ -874,37 +874,6 @@ else
   AC_MSG_RESULT([no])
 fi
 
-AC_MSG_CHECKING([if the high-level tools are enabled])
-AC_ARG_ENABLE([hltools],
-     [AS_HELP_STRING([--enable-hltools],
-                     [Enable the high-level tools.
-                     [default=yes]
-                     ])],
-     [HDF5_HL_TOOLS=$enableval])
-
-if test "X${HDF5_HL}" = "Xyes" -a "X-$HDF5_HL_TOOLS" = "X-yes"; then
- AC_MSG_RESULT([yes])
- HL_TOOLS="tools"
-else
-  AC_MSG_RESULT([no])
-fi
```

Comment:
This option was added to disable the GIF tools,. Now that we have a specific option for that, there's no reason to have a separate option for the HL tools.